wReleaseNotes/2024-06_ReleaseNotes.md

```diff
@@ -3,19 +3,48 @@ title: Release Notes
 filename: "2024-06_ReleaseNotes.md"
 layout: custom
 keywords: [change log, updates, versions, history, what's new]
-headings: ["June 2024", "Documentation Updates"]
-links: []
+headings: ["June 2024", "Excel Addin v2.5.1.1", "Portal Site", "Documentation Updates"]
+links: ["/wAbout/SingleUser.html", "/wAbout/SharedComputer.html", "/wPortal/Roles.html", "/wPortal/FolderPermissions.html"]
 image_dir: ""
 images: []
 description: Release notes for this month's updates
 ---
 * * *
 
-## May 2024
+## June 2024
+
+### Excel Addin v2.5.1.1
+
+_Released 6/4/24_
+
+- 🐞 Wrapped decimal conversion in try/catch where excel versions were being read in addin manager. This fixes a bug where setting the region on your machine causes the addin manager to not open.
+
+- ✅ The main settings and login information are moved from the AppData local to AppData roaming so that when a user is in a shared environment with multiple possible windows instances they could log into, their Interject login and settings can roam between them. This will have little to no effect on single user computers as the roaming folder always exists and persists between logins.
+
+- ✅ Adds a hook to sanitize incoming logs for potentially sensitive data that is included in parameters in error logs or database errors controlled by third party, not reports. Currently only looking for SSNs.
+
+- ✅ The logging feature gathers a minimal amount of data and sends it to the Platform API in batches of no more than 100 or if the last logged data was added more than 5 hours ago.
+
+- ✅ No more than once per day, the add-in makes a single call to the Platform API to gather which log categories it should be logging. This only happens if it hasn't checked in more than 24 hours. That is also a way to turn off logging for all users; by setting the endpoint to return an empty string, no logs will be gathered.
```

Comment:
sometimes we have add-in and sometimes addin. it should be one or the other. I see addin a lot more, so probably that (even though it's technically wrong)

Reply:
You're right, it should all be Addin for consistency, although some day we might want to switch it to Add-in